cmd/api/src/go.mod

```diff
@@ -47,7 +47,9 @@ require (
 	github.com/unrolled/secure v1.13.0
 	github.com/zenazn/goji v1.0.1
 	go.uber.org/mock v0.2.0
+	go.uber.org/zap v1.13.0
 	golang.org/x/crypto v0.18.0
+	golang.org/x/exp v0.0.0-20230515195305-f3d0a9c9a5cc
```

Comment:
are these intentionally added? I've seen these in other PRs as well so curious if this is need or should be in a different PR?